docs/050-breaking-changes.rst

```diff
@@ -0,0 +1,379 @@
+********************************
+Solidity v0.5.0 Breaking Changes
+********************************
+
+This section highlights the main breaking changes introduced in Solidity
+version 0.5.0, along with the reasoning behind the changes and how to update
+affected code.
+For the full list check
+`the release changelog <https://github.com/ethereum/solidity/releases/tag/v0.5.0>`_.
+
```

Comment:
Perhaps we should add a note that 0.5.0-contracts can still interface with contracts and even libraries compiled with older versions of Solidity. The only change would be to slightly modify the interfaces to include data locations and visibility specifiers.